freestyle-logging/jvm/src/main/scala/loggingJVM.scala

```diff
@@ -19,35 +19,84 @@ package freestyle
 import cats.MonadError
 import freestyle.logging._
 import journal._
+import org.slf4j.LoggerFactory
 
 object loggingJVM {
 
   object implicits {
-    implicit def freeStyleLoggingHandler[M[_], C: Manifest](
+    implicit def freeStyleLoggingHandler[M[_]](
         implicit ME: MonadError[M, Throwable]): LoggingM.Handler[M] =
       new LoggingM.Handler[M] {
 
-        val logger = Logger[C]
+        val logger = Logger("")
```

Comment:
It might be possible to pass in the Manifest/ClassTag in each of corresponding methods through the implicit parameter list.